exporter/clickhouseexporter/config.go

```diff
@@ -52,6 +54,21 @@ type Config struct {
 	// Ignored if async inserts are configured in the `endpoint` or `connection_params`.
 	// Async inserts may still be overridden server-side.
 	AsyncInsert bool `mapstructure:"async_insert"`
+	// MetricsTables defines the table names for metric types.
+	MetricsTables MetricTablesConfig `mapstructure:"metrics_tables"`
```

Comment:
Can we mark `MetricsTableName` as deprecated in favor of this (https://go.dev/wiki/Deprecated)? Can we also add a log telling users that option is deprecated?